common/colour/colour-fr.html

```diff
@@ -0,0 +1,314 @@
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md">
+		<h5 class="mrgn-tp-md">Danger</h5>
+		<p class="text-danger">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	</div>
+</div>
+<h4>Exemple de code</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;En sourdine&lt;/h5&gt;
+		&lt;p class="<strong>text-muted</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Primaire&lt;/h5&gt;
+		&lt;p class="<strong>text-primary</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Succès&lt;/h5&gt;
+		&lt;p class="<strong>text-success</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Info&lt;/h5&gt;
+		&lt;p class="<strong>text-info</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Avertissement&lt;/h5&gt;
+		&lt;p class="<strong>text-warning</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+	&lt;div class="well mrgn-tp-md mrgn-bttm-md"&gt;
+		&lt;h5 class="mrgn-tp-md"&gt;Danger&lt;/h5&gt;
+		&lt;p class="<strong>text-danger</strong>"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3>Alignement du texte</h3>
+<p>Aligner le texte sur la page.</p>
+<h4>Exemple</h4>
+<div class="well">
+	<div class="well mrgn-tp-md mrgn-bttm-md text-left">
+		<h5 class="mrgn-tp-md">Aligné à gauche</h5>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-right">
+		<h5 class="mrgn-tp-md">Aligné à droite</h5>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-center">
+		<h5 class="mrgn-tp-md">Aligné au centre</h5>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-justify">
+		<h5 class="mrgn-tp-md">Aligné-justifier</h5>
+		<p>Ambitioni dedisse scripsisse iudicaretur. Cras mattis iudicium purus sit amet fermentum. Donec sed odio operae, eu vulputate felis rhoncus. Praeterea iter est quasdam res quas ex communi. At nos hinc posthac, sitientis piros Afros. Petierunt uti sibi concilium totius Galliae in diem certam indicere. Cras mattis iudicium purus sit amet fermentum.</p>
+	</div>
+	<div class="well mrgn-tp-md mrgn-bttm-md text-nowrap" style="width: 8rem; border: 2px solid black">
+		<h5 class="mrgn-tp-md">Pas d'emballage</h5>
```

Comment:
"Emballage" n'est pas le terme approprié ici. Voir les mots alternatifs sur Termium Plus.

Reply:
Addressed in commit "Addressed Review Comments"